File: 02_representation/base64/decode.go
package main

import (
	"encoding/base64"
	"fmt"
)

func main() {
	b64 := "SGVsbG8gd29ybGQsIG1hYyB3YXMgaGVyZQ=="
	bytes, err := base64.StdEncoding.DecodeString(b64)
	if err != nil {
		panic(err)
	}
	text := string(bytes)
	fmt.Println(text)
}

File: 02_representation/base64/decode.cs
var b64 = "SGVsbG8gd29ybGQsIG1hYyB3YXMgaGVyZQ==";
var bytes = Convert.FromBase64String(b64);
var text = System.Text.Encoding.UTF8.GetString(bytes);
Console.WriteLine(text);
